app/javascript/react/screens/App/Overview/components/Migrations/MigrationsInProgressCard.js

```diff
@@ -19,14 +19,138 @@ const MigrationsInProgressCard = ({ plan, allRequestsWithTasks, reloadCard, hand
   const requestsOfAssociatedPlan = allRequestsWithTasks.filter(request => request.source_id === plan.id);
   const mostRecentRequest = requestsOfAssociatedPlan.length > 0 && getMostRecentRequest(requestsOfAssociatedPlan);
 
+  // === TODO FIXME-- THE BELOW IS MOCK MUTATION CODE TO BE REMOVED ===
+  // This code:
+  // * sticks a fake active pre-playbook in the first migration on the screen
+  // * sticks a fake active post-playbook in the third migration on the screen
+  const dga = window.dangerousGlobalAccumulator;
+  const mockTasks = mostRecentRequest.miq_request_tasks.map((task, i) => ({
+    ...task,
+    options: {
+      ...task.options,
+      playbooks: {
+        pre: {
+          job_id: 4,
+          status: dga === 0 && i === 0 ? 'Active' : 'Succeeded',
+          last_task: 'Task name'
+        },
+        post: {
+          job_id: 5,
+          status: dga === 2 && i === 0 ? 'Active' : 'Succeeded',
+          last_task: 'Task name'
+        }
+      }
```

Comment:
From pre/post playbooks email thread, the playbooks hash will have the following shape

```
{
  "playbooks": {
    "pre": {
      "job_state": "finished",
      "job_status": "Ok",
      "job_id": 4
    },
    "post": {
      "job_state": "pending"
    }
}
```

with the following possible values
- job_state: pending, active, finished
- job_status: Ok, Error